packages/contracts-bedrock/src/L2/L2StandardBridge.sol

```diff
@@ -100,6 +100,7 @@ contract L2StandardBridge is StandardBridge, ISemver {
         virtual
         onlyEOA
     {
+        require(isCustomGasToken() == false, "L2StandardBridge: not supported with custom gas token");
```

Comment:
Should we also mention this is going to be deprecated in the future, as in specs?
```suggestion
        // These methods are deprecated and subject to be removed in the future.
        require(isCustomGasToken() == false, "L2StandardBridge: not supported with custom gas token");
```

Reply:
I added these comments to the natspec